src/hbbft_acs.erl

```diff
@@ -257,8 +257,7 @@ sort_bba_msgs(Msgs) ->
 -spec serialize(acs_data()) -> #{atom() => #{} | binary()}.
 serialize(#acs_data{done=Done, n=N, f=F, j=J, rbc=RBCMap, bba=BBAMap}) ->
     M = #{done => Done, n => N, f => F, j => J},
-    M1 = maps:map(fun(_K, V) -> term_to_binary(V, [compressed]) end, M),
-    M1#{rbc => serialize_state(RBCMap, rbc),
+    M#{rbc => serialize_state(RBCMap, rbc),
```

Comment:
Could be in a single map construction, although maybe the compiler optimizes to that?